dogen/plugins/dist_git.py

```diff
@@ -76,13 +114,14 @@ def repo_info(path):
 
         return name, branch, commit
 
-    def __init__(self, log, source, path, noninteractive=False):
+    def __init__(self, log, output, source, repo, branch, noninteractive=False):
         self.log = log
-        self.path = path
-        self.dockerfile = os.path.join(self.path, "Dockerfile")
+        self.output = output
+        self.repo = repo
+        self.branch = branch
```

Comment:
It seems you do not log branch name anywhere - i think it should be visible in log as it is pretty important part.